offline/k8ssandra_setup.md

```diff
@@ -0,0 +1,201 @@
+```
+
+## [2] Install cert-manager
+Note: For wire-in-a-box setup, cert-manager pods should not be running on same node as ingress-nginx-controller. Hence, we should first install the ingress-nginx-controller and than install cert-manager.
+
+```
+cp ./values/ingress-nginx-controller/prod-values.example.yaml ./values/ingress-nginx-controller/values.yaml
+d helm install ingress-nginx-controller ./charts/ingress-nginx-controller --values ./values/ingress-nginx-controller/values.yaml
+```
+Now check where ingress-nginx-controller pod is running and install cert-manager on a different node.
+
+```
+d kubectl get pods -l app.kubernetes.io/name=ingress-nginx -o=custom-columns=NAME:.metadata.name,NODE:.spec.nodeName
+```
+For e.g. .. if it is `kubenode1`, taint the node
+```
+d kubectl cordon kubenode1
+```
+Now, download cert manager, and place it in the appropriate location:
+```
+wget https://charts.jetstack.io/charts/cert-manager-v1.13.2.tgz
+tar -C ./charts -xvzf cert-manager-v1.13.2.tgz
+```
+
+Install `cert-manager` into a new namespace `cert-manager-ns`.
+```
+d kubectl create namespace cert-manager-ns
+d helm upgrade --install -n cert-manager-ns --set 'installCRDs=true' cert-manager charts/cert-manager
+```
+
+Uncordon the node you cordonned earlier:
+```
+d kubectl uncordon kubenode1
+```
+
+## [3] Install Minio
+Minio and minio-external chart should have been already installed, if you are following the docs_ubuntu_22.04.md
+
+## [4] Deploy K8ssandra Operator
+```
+d helm install k8ssandra-operator charts/k8ssandra-operator -n database --create-namespace
+```
+
+## [5] Configure Minio Bucket for Backups
+Create a secret to access Minio, make a new file `minio-secret.yaml`, and put into it -
+
+Get the aws_access_key_id and aws_secret_access_key from ansible/inventory/offline/group_vars/all/secrets.yaml file, they will be under minio_access_key and minio_secret_key
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+ name: medusa-bucket-key
+ namespace: database
+type: Opaque
+stringData:
+ credentials: |-
+   [default]
+   aws_access_key_id = UIWEGQZ53qVlLuQ2mkM3 #update this
+   aws_secret_access_key = dpZqqiR0Bwz6Kc6J8ruPfTC1VqIPI4EM0Id6TLWG83 #update this
+```
+
+and apply this -
+
+```d kubectl apply -f minio-secret.yaml```
+
+Now, put this medusa config under spec section in ```charts/k8ssandra-test-cluster/templates/k8ssandra-cluster.yaml``` file -
+```
+medusa:
+    storageProperties:
+      storageProvider: s3_compatible
+      region: eu-west-1
+      bucketName: k8ssandra-backups
+      host: minio-external
+      port: 9000
+      prefix: dc1
+      storageSecretRef:
+        name: medusa-bucket-key
+      secure: false
+      maxBackupAge: 7
+```
+
+## Install K8ssandra Test Cluster
+In ```charts/k8ssandra-test-cluster/templates/k8ssandra-cluster.yaml``` update the following variables - 
+
+Set `size` to 1
+
+Set `storageClassName` to "openebs-hostpath"
```

Comment:
Can this not be updated in `charts/k8ssandra-test-cluster/values.yaml`?

Reply:
yep, made the changes upstream to create a new variable for that - https://github.com/wireapp/wire-server/pull/4011

Reply:
fixed